eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubConsumerOptions.java

```diff
@@ -193,55 +160,37 @@ public Retry retry() {
     }
 
     /**
-     * Gets the priority for this receiver. If {@link Optional#isPresent()} is {@code false}, then this is not an
-     * exclusive receiver. Otherwise, it is and there can only be one receiver per (partition + consumer group)
-     * combination.
+     * Gets the owner level for this consumer. If {@link Optional#isPresent()} is {@code false}, then this is not an
+     * exclusive consumer. Otherwise, it is an exclusive consumer, and there can only be one active consumer for each
+     * partition and consumer group combination.
      *
-     * @return An optional priority for this receiver.
+     * @return An optional owner level for this consumer.
      */
-    public Optional<Long> exclusiveReceiverPriority() {
+    public Optional<Long> ownerLevel() {
```

Comment:
I'm not convinced `Optional` will add value here.

Reply:
Removed. :D